deployment/kubernetes/helm/bevrand/templates/frontend-deployment.yaml

```diff
@@ -22,12 +22,12 @@ spec:
       containers:
       - env:
         - name: NODE_ENV
-          value: development
-        image: bevrand/frontend:production
+          value: production
+        image: bevrand/multivuerse:production
         imagePullPolicy: Always
         name: {{ include "bevrand.name" . }}-frontend
         ports:
-        - containerPort: 5000
+        - containerPort: 3000
```

Comment:
Why are we using 3000 for this container, since we're already using the port 5000 convention?

Reply:
Frontend framework convention to use port 3000. Would prefer this convention over our own.